__tests__/app.test.js

```diff
@@ -39,4 +39,41 @@ describe("app", () => {
       });
     });
   });
+
+  describe("api/articles/:article_id", () => {
+    describe("GET", () => {
+      test("Status 200 - responds with an article object matching the article_id requested", () => {
+        return request(app)
+          .get("/api/articles/1")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body).toEqual({
+              article_id: 1,
+              title: "Living in the shadow of a great man",
+              topic: "mitch",
+              author: "butter_bridge",
+              body: "I find this existence challenging",
+              created_at: `2020-07-09T20:11:00.000Z`,
+              votes: 100,
+            });
```

Comment:
this testing is quite brittle, consider refactoring to assert properties exist. SPOILER you'll be adding to this object in a future ticket

Reply:
I will still want specific values for the fields though correct - as this is a single object. So I won't be using the expect.any syntax just the objectContaining.